network/src/counters.rs

```diff
@@ -74,7 +102,7 @@ pub static LIBRA_NETWORK_DISCOVERY_NOTES: Lazy<IntGaugeVec> = Lazy::new(|| {
 pub static LIBRA_NETWORK_RPC_MESSAGES: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
Do we want this? It might make more sense to make a single counter for Rpc Request, Rpc Response, and Direct Send

We can also merge this with the bytes below by an additional field ... metric (bytes versus count)

It would also be great to have a per peer counter for each of these (another field)

Reply:
So, I did a look into this to combine the RPC & DirectSend messages.  The problem is we have another dimension for RPC (the RPC command), so it becomes awfully complex to have it combined in that way.

We can totally add more dimensions to the counters, this was more of a "naming" and "documentation" change to prevent the explosion of changes in this PR.  Those can follow :)